load_balancer.go

```diff
@@ -117,36 +101,6 @@ func (odbi *ovnDBImp) lbDelImp(name string) (*OvnCommand, error) {
 		Table: tableLoadBalancer,
 		Where: []interface{}{condition},
 	}
-
-	// Also delete references from Logical switches
```

Comment:
Please keep this logic. It is the purpose of the PR https://github.com/eBay/go-ovn/pull/33, and it is still correct.

Reply:
ok I thought you wanted LBdel to do LB del only. Will restore